src/app.js

```diff
@@ -0,0 +1,22 @@
+import pkg from '@slack/bolt';
```

Comment:
You can import the `App` directly from the bolt:
```suggestion
import { App } from '@slack/bolt';
```
This removes the need to destructure the `pkg` later (`const { App } = pkg;`)